docs/_cli_docs/docs.md

```diff
@@ -102,7 +102,7 @@ cd example-app
 npm install
 ```
 
-> Note: there are plenty of templates & example apps to choose from! [View all Example Apps here.](https://github.com/zapier/zapier-platform/tree/master/example-apps)
+> Note: there are plenty of templates & example apps to choose from! [View all Example Apps here.](https://github.com/zapier/zapier-platform/wiki/Example-Apps)
```

Comment:
Not for this PR, but I wonder if the wiki should really be a README for the [example-apps package](https://github.com/zapier/zapier-platform/tree/master/example-apps)

Reply:
Makes sense! I created this Jira ticket for the backlog: https://zapierorg.atlassian.net/browse/PDE-2026